.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+# https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        node: ['8.0', '8', '10.0', '10', '12.0', '12', '14.0', '14', '15']
```

Comment:
Just noting the (preexisting) slightly different version pattern here of testing both the `.0` of a major release release as well as the latest.